packages/webdriverio/src/utils/findStrategy.ts

```diff
@@ -28,6 +28,9 @@ const IMAGEPATH_MOBILE_SELECTORS_ENDSWITH = [
     '.jpg', '.jpeg', '.gif', '.png', '.bmp', '.svg'
 ]
 
+export const UPPER_CASE_SYMBOLS = 'ABCDEFGHIJKLMNOPQRSTUVWXYZÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞŸĞİŞАБВГДЕЁЖЗИЙКЛМНОПРСТУФХЦЧШЩЪЫЬЭЮЯ'
+export const LOWER_CASE_SYMBOLS = 'abcdefghijklmnopqrstuvwxyzàáâãäåæçèéêëìíîïðñòóôõöøùúûüýþÿğışабвгдеёжзийклмнопрстуфхцчшщъыьэюя'
```

Comment:
```suggestion
export const UPPER_CASE_SYMBOLS = '\p{Lu}'
export const LOWER_CASE_SYMBOLS = '\p{Ll}'
```

Reply:
sorry, but the `translate` function doesn't support such way - https://developer.mozilla.org/en-US/docs/Web/XML/XPath/Reference/Functions/translate (I checked it out). In the browser context we can use only xPath@1x

Reply:
Thanks for checking. One last request: can we add a comment above these two variables, documenting why we are including all these characters and that we have evaluated my suggestion and can't use it for this reason? Making sure that people understand the context and don't have to dig through GitHub PRs.

Reply:
yeah, sure